app/controllers/favorites_controller.rb

```diff
@@ -1,7 +1,9 @@
 class FavoritesController < ApplicationController
 
   def index
-    @pets = Pet.find(session[:favorites])
+    if !session[:favorites].nil?
```

Comment:
This could be re-written to be a little more straight-forward as `session[:favorites].present?`

Reply:
Aw that's better